### ISSUE TITLE
SQLite support

### ISSUE THREAD

**Issue by gedw99** [CECTC/dbpack#256]
Litefs is a golang compliment to SQLite. It supports master slave SQLite where the changes on the master are saved to S3 and then propagated to the slaves.

A proxy that is able to distinguish writes / reads would be complimentary because then writes can go to the master db and reads to the slave db.

https://github.com/superfly/litefs

as I understand it the proxy does not support SQLite protocol ?

**Comment by dk-lockdown** [comment]
Yes, we do not support the SQLite protocol yet. If necessary, we hope that someone in the community can work with us to complete the support for SQLite.

**Comment by gedw99** [comment]
Might be worth collaborating with Ben as it looks like he looks to write a proxy. There are a few issues alluding to it.

i
